src/lib.rs

```diff
@@ -80,19 +90,20 @@ ast_struct! {
         pub generics: Generics,
         pub paren_token: token::Paren,
         pub inputs: Punctuated<FnArg, Token![,]>,
+        pub variadic: Option<Variadic>,
         pub output: ReturnType,
-        pub block: Block,
     }
 }
```

Comment:
I suggest you get rid of this `Signature` and simply reexport it from `syn`.

Reply:
IIRC, [`syn::Signature`](https://docs.rs/syn/1.0.11/syn/struct.Signature.html) requires `full` feature of `syn`.

Reply:
Yeah, that's quite sad...